src/stores/config.js

```diff
@@ -14,7 +14,7 @@ exports.setUp = (basePath, blobStore, locks) => {
           }
           var result
           try {
-            result = JSON.parse(config)
+            result = JSON.parse(config.toString())
```

Comment:
Actually I know, local storage converts everything to strings, as it can't store blobs, so that's why that worked is my guess